docs/installing.rst

```diff
@@ -6,6 +6,17 @@ Fathom consists of 3 parts. Here's how to install each one.
 
 .. _fathomfox-installation:
 
+Commandline Tools
+=================
+
+Fathom's commandline tools take your labeled pages as input and train the machine-learning model. They also contain the simplest way to collect pages, FathomFox. If you don't already have Python 3.7 or better, download it from https://www.python.org/downloads/. Then, install the tools by running... ::
```

Comment:
"Contain" a "way" sounds funny to my ear. Maybe we could just say "They also contain an embedded copy of FathomFox (see below)."